### ISSUE TITLE
Allow is not in 13.5.2 (Trac #53)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#53]
```text
#!html
            <p>Section 14.7 states:</p>            
            <p>"A proxy MUST NOT modify the Allow header field even if it does not
            understand all the methods specified, since the user agent might have other
            means of communicating with the origin server."</p>            
            <p>However, section 13.5.2 (Non-modifiable Headers) makes no mention of Allow. This seems like an error, but I'm not entirely sure what the fix should be -- remove 13.5.2 and push the (not-)modifiable information in the definition of the respective headers, or to maintain 13.5.2 in parallel with all of the header definitions, or to push all the information out of the header definitions into 13.5.2.</p>            
            <p>The easy fix for now would be to just make a mention of Allow in 13.5.2.</p>
            <p>Additionally, Server should also be included.</p>
```

Migrated from https://trac.ietf.org/ticket/53
```json
{
    "status": "closed",
    "changetime": "2011-03-30T16:40:45",
    "_ts": "1301503245000000",
    "description": "{{{\n#!html\n            <p>Section 14.7 states:</p>            \n            <p>\"A proxy MUST NOT modify the Allow header field even if it does not\n            understand all the methods specified, since the user agent might have other\n            means of communicating with the origin server.\"</p>            \n            <p>However, section 13.5.2 (Non-modifiable Headers) makes no mention of Allow. This seems like an error, but I'm not entirely sure what the fix should be -- remove 13.5.2 and push the (not-)modifiable information in the definition of the respective headers, or to maintain 13.5.2 in parallel with all of the header definitions, or to push all the information out of the header definitions into 13.5.2.</p>            \n            <p>The easy fix for now would be to just make a mention of Allow in 13.5.2.</p>\n            <p>Additionally, Server should also be included.</p>\n}}}",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "incorporated",
    "time": "2007-12-22T02:35:06",
    "component": "p1-messaging",
    "summary": "Allow is not in 13.5.2",
    "priority": "normal",
    "keywords": "",
    "milestone": "14",
    "owner": "",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20080104T06:10:46: 
* mnot@pobox.com changed component from "" to "messaging"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080105T03:45:06: fielding@gbiv.com changed milestone from "unassigned" to "02"

**Comment by mnot** [comment]
Trac update at 20080228T05:26:35: mnot@pobox.com changed milestone from "02" to "03"

**Comment by mnot** [comment]
Trac update at 20080717T14:22:49: julian.reschke@gmx.de changed milestone from "03" to "04"

**Comment by mnot** [comment]
Trac update at 20080829T08:46:35: julian.reschke@gmx.de changed milestone from "04" to "unassigned"

**Comment by mnot** [comment]
Trac update at 20090324T16:10:41: julian.reschke@gmx.de commented:

> Note: the related text is gone from P6. We probably should create a new issue for the current TODO in P6 (end-to-end and hop-by-hop headers, non-modifiable headers removed; re-spec in p1) and then close this issue, noting the new test in P1 is supposed to fix this issue.

**Comment by mnot** [comment]
Trac update at 20110330T16:39:11: julian.reschke@gmx.de commented:

> From [1245]:
> 
> Add "Allow" and "Server" to the list of headers not to be modified by proxies (see #53)

**Comment by mnot** [comment]
Trac update at 20110330T16:40:45: 
* julian.reschke@gmx.de changed milestone from "unassigned" to "14"
* julian.reschke@gmx.de changed priority from "" to "normal"
* julian.reschke@gmx.de changed resolution from "" to "incorporated"
* julian.reschke@gmx.de changed severity from "" to "Active WG Document"
* julian.reschke@gmx.de changed status from "new" to "closed"
